src/components/Navigation/index.js

```diff
@@ -1,37 +1,60 @@
 import React from 'react';
+import Button from '../Button';
 import lines from "../../images/lines.png";
 import shirt from "../../images/shirt.png";
 import './style.css';
+import IconButton from '../IconButton';
+import arrow from '../../images/arrow.png';
 
+const navBtn = ['Техника с доставкой', 'Телефоны', 'Недвижимость', 'Легковые авто', 'Велосипеды',
+  'Работа', 'Платья', 'Строительный инструмент', 'Садовая техника', 'Запчасти,расходники',
+'Коляски', 'Диваны', 'Холодильники'];
 
 const Navigation = () =>(
     <nav className="navigation">
-      <button className="btn--all-categories">
-        <img className="btn--all-categories__img" src={lines} alt="Lines"/>
-        <p className="btn--all-categories__text">Категории</p>
-      </button>
+      <Button
+        className="btn_category_primary "
+        status="default_blue"
+        icon={lines}
+        alt="Lines"
+        label="Категории"
+        labelSize="large"
+        iconSize="primary"
+        bold={true}
+      />
       <div className="scroll-panel-nav">
-        <a className="nav--category">
-          <img className="nav-img--skirt" src={shirt} alt="Shirt"/>
-          <p>Модное лето</p>
-        </a>
-        <a className="nav--category">Техника с доставкой</a>
-        <a className="nav--category">Телефоны</a>
-        <a className="nav--category">Недвижимость</a>
-        <a className="nav--category">Легковые авто</a>
-        <a className="nav--category">Велосипеды</a>
-        <a className="nav--category">Работа</a>
-        <a className="nav--category">Платья</a>
-        <a className="nav--category">Строительный инструмент</a>
-        <a className="nav--category">Садовая техника</a>
-        <a className="nav--category">Запчасти,расходники</a>
-        <a className="nav--category">Коляски</a>
-        <a className="nav--category">Диваны</a>
-        <a className="nav--category">Холодильники</a>
+        <Button
+          className="nav-category "
+          icon={shirt}
+          alt="Shirt"
+          label="Модное лето"
+          iconSize="primary"
+          labelSize="large"
+          status="default"
+        />
+        {navBtn.map(text => (
```

Comment:
Not as a change request but as a thing to thinks about...

ButtonGroup component :)